frontend/fingertips-frontend/package.json

```diff
@@ -72,6 +74,8 @@
     "npm": ">=10.9.0"
   },
   "overrides": {
+    "react": "$react",
+    "react-dom": "$react-dom",
```

Comment:
Do we need these overrides any more? We're now on the stable release of React 19 so hopefully the `highcharts-react-official` peer dependency is fulfilled by that.

Reply:
I've tried this, and I think they can be removed.